components/Feed.tsx

```diff
@@ -139,6 +161,7 @@ function Feed({
       }}
     >
       {renderedStats}
+      {renderedVideoBitrate}
```

Comment:
I don't think creating these variables are necessary. Calling `{renderStats()}` and `{renderVideoBitrate()}` here is probably more readable.

You can move the functions inside a `useCallback` if you're worried about performance.

Reply:
Resolved in [00240b9](https://github.com/team-telnyx/telnyx-meet/pull/58/commits/00240b995f39346566cb304449eef706802844d2)